src/accounts.cs

```diff
@@ -87,6 +87,8 @@ interface ACAccountStore {
 		[Async]
 		void RequestAccess (ACAccountType accountType, ACRequestCompletionHandler completionHandler);
 
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Public notification deprecated. None Apple applications should not use this API.")]
+		[Deprecated (PlatformName.MacOSX, 10, 16, message: "Public notification deprecated. None Apple applications should not use this API.")]
```

Comment:
The second sentence is not grammatically correct, and I don't think we need it anyway, so I think we can just remove it:

```suggestion
		[Deprecated (PlatformName.iOS, 14, 0, message: "Public notification is deprecated.")]
		[Deprecated (PlatformName.MacOSX, 10, 16, message: "Public notification is deprecated.")]
```